testing/integration_test.go

```diff
@@ -54,14 +55,14 @@ set -eo pipefail
 # Display commands being run
 set -x
```

Comment:
Any reason there isn't
```
# Fail on any error.
set -eo pipefail
```
like in https://code.googlesource.com/gocloud/+/master/profiler/integration_test.go#52?

Reply:
I think it's there, on lines 51-54.